jetty-websocket/javax-websocket-tests/src/test/java/org/eclipse/jetty/websocket/javax/tests/server/EndpointViaConfigTest.java

```diff
@@ -126,6 +126,10 @@ public void testEcho() throws Exception
             URI uri = wsb.getWsUri();
 
             WebAppContext webapp = wsb.createWebAppContext();
+            // TODO It's not great that we have to expose these APIs for testing as it can hide other classpath issues!
+            webapp.getSystemClassMatcher().include("org.eclipse.jetty.websocket.javax.");
+            webapp.getServerClassMatcher().exclude("org.eclipse.jetty.websocket.javax.");
```

Comment:
This wasn't needed before, why now?
And can't this just be a WebSocketConfiguration instead?
Or a JavaxWebSocketConfiguration?

Reply:
This seems like it would be a requirement for embedded-jetty users now too.
Is that true?

Reply:
We didn't have to expose them before because previously JavaxWebSocketConfiguration.java always exposed them.
Ideally testing would be in a different package all together and within the webapp.

embedded users will probably not be writing code in the org.eclipse.jetyt.websocket namespace, so they should be unaffected.

Reply:
I still don't follow this statement.

Create a testcase on `com.acme.websocket` or `org.eclipse.jetty.test` that uses a `WebAppContext` that has `javax.websocket` usage in embedded mode **without** the need to manipulate the Server/System classes.

Reply:
@joakime The problem is that our test endpoints are in `org.eclipse.jetty...`  So currently we have to open the classloader so they can be seen.   If we moved them to `com.acme...`  or some package that was not hidden by default, then we would not have to open the classpath... which would be best as doing so may make the tests work for unexpected reasons and thus hide other problems.

@lachlan-roberts - can you fix this? Start by removing these include/excludes from the tests, you will then see the class not found exceptions that will tell  you want classes need to be moved.   You may be better off making a test webapp in a separate module.